root/programs/C#/Frameworks/Infrastructure/Public/IO/SymmetricCryptography.cs

```diff
@@ -438,9 +438,9 @@ private static SymmetricAlgorithm CreateSymmetricAlgorithm(EnumSymmetricAlgorith
         {
             SymmetricAlgorithm sa = null;
 
-            // AesCryptoServiceProvider, AesManagedは3.5からの提供。
-            // 暗号化プロバイダ選択のポリシー（Managed → CSP → CNG）
-            // に合わせて、AesCryptoServiceProviderを削除。
+            // AesCryptoServiceProvider, AesManagedは.NET Framework 3.5からの提供。
+            // 暗号化プロバイダ選択の優先順は、Managed → CAPI(CSP) → CNG。
```

Comment:
@yusukemaegawa 

Windows 2003 and XP also fall within the scope of support for the previous .NET 2.0 - 3.5 support range. So, I had been decided not to adopt CNG adoption.
Now, since the support range has reached .NET 4.6 or later, the support range OS has become Vista or later. Therefore, in the future, I will switch from CAPI to CNG.